.travis.yml

```diff
@@ -102,7 +102,7 @@ jobs:
               # Enable emulation layer
               - export USE_HB_EMUL=1
               # Build and install pytorch
-              - python setup.py install
+              - python setup.py develop
```

Comment:
I made the same change. but let's keep it here. Otherwise I think travis will error out due to log being too long.